docs/boostrap-as-non-root-user.md

```diff
@@ -0,0 +1,9 @@
+# Bootstrapping RPC-CEPH as non-root user
+
```

Comment:
Might want to add a note in here that at this time you cannot run the setup of an AIO as a non-root user.